README.md

```diff
@@ -53,6 +61,8 @@ Your skills contribution pull requests must include the following:
 > There is a limit to how much content can exist in the question/answer pairs for the model to process. Due to this, only add a maximum
 > of around 2300 words to your question and answer seed example pairs in the `qna.yaml` file.
 
+Compositional skills can either be grounded (includes a context) or ungrounded (does not include a context).  Grounded or ungrounded is the last node in the taxonomy tree, for example: `linguistics/writing/poetry/haiku/ungrounded` or `linguistics/grammar/grounded`. The `qna.yaml` is in this final node.
```

Comment:
Again, I am unclear on the utility of putting grounded/ungrounded in the path (since the qna.yaml can have or not have context independent of the path).